### ISSUE TITLE
Error on nested  malformed list <ol> <ul> objects: Cannot read properties of null (reading 'querySelector')  

### ISSUE THREAD

**Issue by wvddrss** [froala/react-froala-wysiwyg#401]
```
<ol>
	<ol>
		<ol>
			<li><span>item 1</span></li>
			<li><span>item 2</span></li>
		</ol>
           </ol>
</ol>
```

It throw:

```
Uncaught TypeError: Cannot read properties of null (reading 'querySelector')
    at s (froala_editor.pkgd.min.js:7:25822)
    at r (froala_editor.pkgd.min.js:7:26299)
    at Object.g [as lists] (froala_editor.pkgd.min.js:7:26303)
    at h (froala_editor.pkgd.min.js:7:130299)
    at Object.L [as set] (froala_editor.pkgd.min.js:7:135815)
    at n (froala_editor.pkgd.min.js:7:523930)
    at Object.E [as toggle] (froala_editor.pkgd.min.js:7:524314)
    at w.Bootstrap.callback (froala_editor.pkgd.min.js:7:525236)
    at Object.n [as exec] (froala_editor.pkgd.min.js:7:44720)
    at c (froala_editor.pkgd.min.js:7:221305)
s @ froala_editor.pkgd.min.js:7
r @ froala_editor.pkgd.min.js:7
g @ froala_editor.pkgd.min.js:7
h @ froala_editor.pkgd.min.js:7
L @ froala_editor.pkgd.min.js:7
n @ froala_editor.pkgd.min.js:7
E @ froala_editor.pkgd.min.js:7
callback @ froala_editor.pkgd.min.js:7
n @ froala_editor.pkgd.min.js:7
c @ froala_editor.pkgd.min.js:7
a @ froala_editor.pkgd.min.js:7
t @ froala_editor.pkgd.min.js:7
d @ froala_editor.pkgd.min.js:7
a @ froala_editor.pkgd.min.js:7
(anonymous) @ froala_editor.pkgd.min.js:7
(anonymous) @ froala_editor.pkgd.min.js:7Understand this error```

**Comment by mikebertiean** [comment]
Same issue.

**Comment by ilyaskarim** [comment]
Fixed in V4.3.1: https://froala.com/wysiwyg-editor/changelog/#4.3.1
